health-check/index.js

```diff
@@ -5,53 +5,117 @@
  * Description : Health check functionality for various services.
  */
 
-const { v1: uuidv1 } = require('uuid');
-const mongodb = require('./services/mongodb');
-const kafka = require('./services/kafka');
-const httpService = require('./services/httpService');
+const { v1: uuidv1 } = require('uuid')
+const httpService = require('./services/httpService')
 
-async function healthCheckHandler(config) {
-	const checks = [];
-	
-	if (config.checks.mongodb?.enabled) {
-		const healthy = await mongodb.check(config.checks.mongodb.url);
-		checks.push(serviceResult('MongoDB', healthy));
+/**
+ * Perform health checks for various services.
+ *
+ * @param {Object} config - The configuration object containing service health settings.
+ * @param {boolean} basicCheck - If true, includes microservice health checks.
+ * @param {string} [currentServiceName=''] - Optional. Name of the current service to avoid self-check.
+ * @returns {Promise<Object>} - A formatted response with health check results.
+ */
+async function healthCheckHandler(config, basicCheck, currentServiceName = '') {
```

Comment:
@MallanagoudaB are we not validating config to make sure required key values are passed